### PR TITLE
Refactor Overview section (Phase 2)

### PR DESCRIPTION
**Pull request by acolleux-microej**

### CODE REVIEW

File: overview/index.rst [hunk 1]
```diff
@@ -1,26 +1,109 @@
...
+-----------
+
+MicroEJ VEE (Virtual Execution Environment) is an application container for resource-constrained embedded devices running on microcontrollers or microprocessors. 
+It allows devices to run multiple and mixed managed code (Java, JavaScript, ...) and C software applications.
+
+MicroEJ VEE provides a fully configurable set of services that can be expanded, including:
+
+- a secure multi-application framework,
+- a GUI framework (includes widgets),
+- a network connection with security (SSL/TLS, HTTPS, REST, MQTT, ...),
```
Comment (gharelMicroEJ) [inline overview/index.rst:26]: I would not use connection here, framework is used in the other bullet points.

File: overview/index.rst [hunk 2]
```diff
@@ -1,26 +1,109 @@
...
+MICROEJ SDK
+-----------
+
+MICROEJ SDK offers a comprehensive toolset to build the embedded software of
+a device. The SDK covers two levels in device software development:
+
+-  Device Firmware development
+-  Application development
+
+The firmware will generally be produced by the device OEM, it includes
```
Comment (gharelMicroEJ) [inline overview/index.rst:48]:
```suggestion
The firmware will generally be produced by the device's OEM, it includes
```

File: overview/index.rst [hunk 3]
```diff
@@ -1,26 +1,109 @@
...
+   :scale: 80%
+
+   MICROEJ VEE Overview
+
+For more information about our Supported Processor Architectures, visit the `Supported Hardware <https://developer.microej.com/supported-hardware/>`_ page.
+
+MICROEJ SDK
+-----------
+
+MICROEJ SDK offers a comprehensive toolset to build the embedded software of
```
Comment (gharelMicroEJ) [inline overview/index.rst:42]:
```suggestion
MICROEJ SDK offers a comprehensive toolset to build the software for embedded devices. 
```

**Review by gharelMicroEJ — CHANGES_REQUESTED** [review]

File: overview/index.rst (hunk 1 above)
Comment (acolleux-microej) [reply overview/index.rst]: Fixed

**Review by frivieremicroej — CHANGES_REQUESTED** [review]
State on Introduction vs Overview

File: overview/executableBuildWorkflow.rst [hunk 4]
```diff
@@ -1,22 +1,22 @@
-.. _tutorialUnderstandMicroEJFirmwareBuild:
+.. _executableBuildWorkflow:
```
Comment (frivieremicroej) [inline overview/executableBuildWorkflow.rst:1]: State on Introduction vs Overview

**Review by frivieremicroej — CHANGES_REQUESTED** [review]

**Pull request merged**